parts/k8s/cloud-init/masternodecustomdata.yml

```diff
@@ -104,15 +104,6 @@ write_files:
   content: !!binary |
     {{CloudInitData "aptPreferences"}}
 
-{{if .MasterProfile.IsUbuntuNonVHD}}
```

Comment:
should we keep the if/end as scaffolding to facilitate development for future CIS files that get added to the Ubuntu distro first?